CONTRIBUTORS.md

```diff
@@ -2,6 +2,7 @@
 
 &nbsp; | &nbsp; | &nbsp;
 :---: | :---: | :---:
+[<img src="https://avatars.githubusercontent.com/u/43273583" width="80" /><br />@alexander-kuruvilla](https://github.com/alexander-kuruvilla)<br />Web Development | [<img src="https://avatars.githubusercontent.com/u/54772437" width="80" /><br />@Sneha65](https://github.com/Sneha65)<br />Web Development | [<img src="https://avatars.githubusercontent.com/u/137500548" width="80" /><br />Venkatesh Sangam](https://github.com/venkatesh2k3)<br />Web Development
```

Comment:
@nya-elimu , my github link is wrong. It is https://github.com/SnehaHS65

Reply:
@SnehaHS65 Fixed: https://github.com/elimu-ai/wiki/commit/8a67d0bf94713478014671ea345a2bda0078b32a